catalog/transformer/catalog-transformer-mgmp/src/main/java/org/codice/alliance/catalog/converter/mgmp/MgmpConverter.java

```diff
@@ -0,0 +1,1230 @@
+        addModifiedDate(pathValueTracker, metacard);
+        addCreatedDate(pathValueTracker, metacard);
+        addExpirationDate(pathValueTracker, metacard);
+    }
+
+    private void addMdIdentificationLanguage(XstreamPathValueTracker pathValueTracker,
+            MetacardImpl metacard) {
+
+        List<String> languageCodes = Utilities.getValues(metacard, Core.LANGUAGE);
+
+        if (languageCodes.isEmpty()) {
+            languageCodes = Collections.singletonList(Locale.ENGLISH.getISO3Language());
+        }
+
+        List<String> languageTexts = languageCodes.stream()
+                .map(code -> code.equals(Locale.ENGLISH.getISO3Language()) ?
+                        Locale.ENGLISH.getDisplayLanguage() :
+                        "")
+                .collect(Collectors.toList());
+
+        List<String> languageCodeList = Utilities.replace(languageCodes, LANGUAGE_CODE_LIST);
+
+        Utilities.addMultiValues(pathValueTracker,
+                languageCodeList,
+                MD_IDENTIFICATION_LANGUAGE_CODE_LIST_PATH);
+        Utilities.addMultiValues(pathValueTracker,
+                languageCodes,
+                MD_IDENTIFICATION_LANGUAGE_CODE_LIST_VALUE_PATH);
+        Utilities.addMultiValues(pathValueTracker,
+                languageTexts,
+                MD_IDENTIFICATION_LANGUAGE_TEXT_PATH);
+
+    }
+
+    private void addMdIdentificationAggregationInfo(XstreamPathValueTracker pathValueTracker,
+            MetacardImpl metacard) {
+        addMdIdentificationAggregationInfoAggregateDataSetIdentifier(pathValueTracker, metacard);
+    }
+
+    private void addMetadataStandardVersion(XstreamPathValueTracker pathValueTracker) {
+        pathValueTracker.add(new Path("/MD_Metadata/metadataStandardVersion/gco:CharacterString"),
+                "2.0");
+    }
+
+    private void addMetadataStandardName(XstreamPathValueTracker pathValueTracker) {
+        pathValueTracker.add(new Path("/MD_Metadata/metadataStandardName/gco:CharacterString"),
+                "MOD Geospatial Metadata Profile");
+    }
+
+    private void addNamespaces(XstreamPathValueTracker pathValueTracker) {
+        pathValueTracker.add(new Path(GMD_NAMESPACE_PATH), GMD_NAMESPACE);
+        pathValueTracker.add(new Path(GCO_NAMESPACE_PATH), GCO_NAMESPACE);
+        pathValueTracker.add(new Path(MGMP_NAMESPACE_PATH), MGMP_NAMESPACE);
+        pathValueTracker.add(new Path("/MD_Metadata/@xmlns:gml"), "http://www.opengis.net/gml/3.2");
+        pathValueTracker.add(new Path("/MD_Metadata/@xmlns:xlink"), "http://www.w3.org/1999/xlink");
+    }
+
+    private void addCharacterSet(XstreamPathValueTracker pathValueTracker) {
+        pathValueTracker.add(new Path(CHARACTER_SET_CODE_LIST_PATH),
+                "http://mod.uk/spatial/codelist/mgmp/2.0/ MGMP_CharacterSetCode");
+        pathValueTracker.add(new Path(CHARACTER_SET_CODE_LIST_VALUE_PATH), "utf8");
+        pathValueTracker.add(new Path(CHARACTER_SET_TEXT_PATH), "UTF-8");
```

Comment:
Recommend having these defined as constants, ENCODING_TYPE versus ENCODING_DESCRIPTION or something to denote that one is the human readable text `UTF-8` versus the programmatic encoding `utf8`.